plugin.video.playonbrowser/resources/language/English/strings.po

```diff
@@ -0,0 +1,78 @@
+# Playon Browser Media Center language file
```

Comment:
Also, something I forgot. Can you change the folder layout as well? See here: https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language